core/src/tangram.cpp

```diff
@@ -379,6 +384,9 @@ bool Map::update(float _dt) {
         }
     }
 
+    if (impl->attribution.isSetup()) {
+        impl->attribution.update(impl->view);
+    }
     FrameInfo::endUpdate();
```

Comment:
I would use `std::unique_ptr<Attribution> attribution`, passing setup parameters to the constructor: when it is set it's already set up